pyproject.toml

```diff
@@ -138,6 +138,7 @@ markers = [
 [tool.mypy]
 show_error_codes = true
 check_untyped_defs = true
+incremental = false
```

Comment:
this pessimizes the experience for local runs, so I don't think we want it.

Reply:
If we have to choose between them, I think we'd prefer to keep incremental. (But I suspect what this really means is that unused config warnings only happen on initial runs, and are ignored on subsequent incremental runs).

Reply:
That's my guess. In any event, we need incremental to remain on. So I would drop this change.